packages/template-mrt-reference-app/app/ssr.js

```diff
@@ -67,6 +67,15 @@ const ENVS_TO_EXPOSE = [
     'aws_lambda_log_stream_name',
     'aws_region',
     'bundle_id',
+    // These "customer" defined environment variables are set by the Manager
+    // and expected by the MRT smoke test suite
+    'customer_defined_array',
+    'customer_defined_boolean',
+    'customer_defined_float',
+    'customer_defined_integer',
+    'customer_defined_json',
+    'customer_defined_null',
+    'customer_defined_string',
```

Comment:
Non-blocking: Do we expect to add are remove keys from this in the future?

If so, you might consider allow listing `customer_*`. This might require a change to `filterAndSortObjectKeys`.

Reply:
Done. The wildcard approach makes sense. I've also removed the /env-vars endpoint since / already returns the env vars.